### PR TITLE
Revisit "Developing LHCb Software"

### PR DESCRIPTION
**Pull request by rmatev**
Preview at http://rmatev.web.cern.ch/rmatev/starterkit/html/first-analysis-steps/lhcb-dev.html

Also I squashed a couple of problems in the build and updated the copyright year.

### CODE REVIEW

File: first-analysis-steps/lhcb-dev.md [hunk 1]
```diff
@@ -14,58 +14,58 @@ similar to what has been
...
- 1. An LHCb-specific workflow using a set of `lb-*` subcommands.  This allows
- you to check out individual packages inside a project, and streamlines the
- modification of a few packages at a time.  (This is closer to the previously
- used `getpack` command.)
+1. A vanilla Git workflow using only the standard Git commands.  This requires
+   you to `clone` and compile an entire LHCb project at a time.
+1. An LHCb-specific workflow using a set of `lb-*` Git subcommands.  This allows
+   you to check out individual packages inside a project, and streamlines the
+   modification of a few packages at a time.  (This is closer to the previously
+   used `getpack` command.)
```
Comment (alexpearce) [inline first-analysis-steps/lhcb-dev.md:34]: I reckon we can delete the references to `getpack` now.

File: first-analysis-steps/lhcb-dev.md [hunk 2]
```diff
@@ -74,13 +74,13 @@ If you want to make changes to a software package, you will need to set up a
 development environment. `lb-dev` is your friend here:
 
 ```bash
-lb-dev --name DaVinciDev DaVinci/v45r1
+lb-dev --name DaVinciDev DaVinci/v45r5
```
Comment (alexpearce) [inline first-analysis-steps/lhcb-dev.md:77]: There is some discussion in #242 about bumping the version. As far as I could tell it's OK in the first-steps lessons, but we should change it coherently.

**Review by alexpearce — COMMENTED** [review]
Nice little changes, thanks :+1:

**Comment by alexpearce** [comment]
The lesson pulls `Phys/StrippingSelections`, but I don't think the students already have any options that use this package, so they can't test any changes very easily.

In the past I've had them checkout `Phys/DecayTreeTuple` or something.

File: first-analysis-steps/lhcb-dev.md [hunk 3]
```diff
@@ -211,82 +245,75 @@ When pushing to a branch in a project in GitLab you will see a message like:
...
 {% callout "Working with a full project checkout" %}
 
-The `lb-git` commands are not strictly necessary, but they're very convenient
+The `git lb-*` commands are not strictly necessary, but they're very convenient
 if you just want to quickly edit one package. Otherwise you'd have to build
 the entire project in which the package is residing, instead of using the
 precompiled version. However, if you develop across multiple packages, or
 want to use more sophisticated `git` commands, nothing prevents you from
-checking out an entire project – just don't be surprised if it takes O(hours)
+checking out an entire project – just don't be surprised if it takes hours
```
Comment (saschastahl) [inline first-analysis-steps/lhcb-dev.md:316]: Maybe we should be more specific here? And say it takes hours on lxplus but if you a have some multi-core machines available it takes less than an hour and only for the first time?

Comment (rmatev) [reply first-analysis-steps/lhcb-dev.md:316]: :+1:

**Review by alexpearce — APPROVED** [review]
LGTM! :+1:

**Pull request merged**